abcix/types/application.go

```diff
@@ -4,8 +4,25 @@ import (
 	context "golang.org/x/net/context"
 )
 
-type MempoolIter interface {
-	// GetNextTransaction(remainBytes int64, remainGas int64, starter []byte) types.Tx
+type mempool interface {
```

Comment:
add a comment noting this is an embedded interface for accessing `mempool.Mempool` interface